docs/iOS-install-and-tutorial-instructions-conda.md

```diff
@@ -0,0 +1,44 @@
+# Installing brainlit on iOS using conda environment:
```

Comment:
Mac, not iOS

Reply:
Changes "iOS" to "macOS" in name of file and headings